src/docs/pages/introduction/getting-started/react.md

```diff
@@ -109,6 +109,107 @@ There are a few cases where native events of Web Components do not behave as exp
   LdInput <code>onChange</code> event invokes when the component loses focus (and the value changed). This is the standard browser behavior but differs from the native React <code>onChange</code> event. Use the <code>onInput</code> event in cases you want to handle user input immediately while typing. Find additional information in the <a href="guides/event-handling/">Event handling guide</a>.
 </ld-notice>
 
+## Cookbook
+
+### React Router
+
+[React Router](https://reactrouter.com/) is a widely used library for routing in client-side react applications. This recipe shows you how to integrate Liquid Oxygen navigation components with React Router. Although the recipe is based on React Router, interacting with other libraries should look quite similar.
+
+Liquid Oxygen provides some sophisticated components for navigation. This recipe uses the following:
+
+- [LdSidenav](/components/ld-sidenav/) is the recommended navigation solution for most applications using Liquid Oxygen. It is flexible, responsive and provides multiple levels of navigation.
+- [LdBreadcrumbs](/components/ld-breadcrumbs/) component is a simple way to display the current location in your application.
+
+As this recipe focuses on how to put together Liquid Oxygen and React Router, we'll only highlight the relevant parts of the code. We assume you already have a basic React Router setup in place. If not, please refer to the [React Router documentation](https://reactrouter.com/en/start/tutorial).
+
+Let's get started with routing. We cannot use the `<Link>` component provided by React Router within LdSidenavNavitem, therefore we use the [useNavigate](https://reactrouter.com/en/main/hooks/use-navigate) hook to navigate to a different route.
+
+```tsx
+// Sidebar.tsx
+//...
+const navigate = useNavigate()
+
+const handleLink = useCallback(
+  (e: React.MouseEvent) => {
+    e.preventDefault()
+    const href = (e.target as HTMLLdSidenavNavitemElement).href
+    if (href) {
+      navigate(href)
+    }
+  },
+  [navigate]
+)
+
+return (
+  // ...
+  <LdSidenavNavitem
+    href={`/mathematical-foundations/discrete-mathematics`}
+    onClick={handleLink}
+  >
+    Discrete Mathematics
+  </LdSidenavNavitem>
+  // ...
+)
+// ...
+```
+
+The `href` prop of `LdSidenavNavitem` tells the component to add `LdLink` which renders a proper anchor tag. This is important for accessibility. As this would already work to navigate but bypass the client-side navigation of React Router, we also add a click handler. The click handler uses the `navigate` function from the `useNavigate` hook to navigate to the specified href. `preventDefault()` is called to prevent the default behavior of the anchor tag.
```

Comment:
No exactly sure about accessibility or seo but for me as a user i like to see in the bottom where a link is linking to so I assume it is best practice to make sure to have an anchor rendered when navigating. 

Now I read what I wrote. Well "This is important for accessibility" actually was a suggestion by copilot ;)

@borisdiakur do you consider that as "important for accessibility" or shall I remove this sentence?

Reply:
Yes. I think you are right, because by default the `ld-sidenav-item` is a `button` and the `href` converts it to a real anchor, so that you not only get the correct announcement from the screen reader, but also can open the link in a new tab etc.